### PR TITLE
Update CSS token per new design system theming capability

### LINKED ISSUE #29: auro-toast: Theming support
### Is your feature request related to a problem? Please describe.

This is regarding theming updates that need to happen to nearly all `auro` components

### Describe the solution you'd like

1. All tokens with the prefix `--auro` need to be renamed to `--ds` so they work with the new WCSS v5 update.
2. The WCSS package needs to be updated to ^5.0.0 and the import paths in the component *.scss files need to be updated as well

### Describe alternatives you've considered

None

### Additional context

Reference this `auro-button` pr https://github.com/AlaskaAirlines/auro-button/pull/231
And this `auro-hyperlink` pr https://github.com/AlaskaAirlines/auro-hyperlink/pull/185

### Exit criteria

All tokens referencing `--auro` have been updated.

### PR DESCRIPTION
**Pull request by irma-kurnia-alaska**
# Alaska Airlines Pull Request

**Fixes:** #29 

## Summary:

- Update the design tokens to reflect the new theming conventions regarding tokens from the AuroDesignTokens and JetstreamDesignTokens repo.

## Type of change:

Please delete options that are not relevant.

- [x] Revision of an existing capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Review by jason-capsule42 — DISMISSED** [review]

File: demo/index.html [hunk 1]
```diff
@@ -23,6 +23,7 @@
       href="https://cdn.jsdelivr.net/npm/prismjs@1.24.1/themes/prism.css"
     />
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@alaskaairux/design-tokens@latest/dist/tokens/CSSCustomProperties.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@aurodesignsystem/design-tokens@latest/dist/tokens/CSSCustomProperties.css">
```
Comment (blackfalcon) [inline demo/index.html]: Please update to mirror this code 

https://github.com/AlaskaAirlines/auro-avatar/blob/main/demo/index.html#L12-L17

File: package.json [hunk 2]
```diff
@@ -25,13 +25,15 @@
   },
   "peerDependencies": {
     "@alaskaairux/design-tokens": "^3.13.0",
```
Comment (blackfalcon) [inline package.json]: Remove legacy package reference

File: package.json [hunk 3]
```diff
@@ -25,13 +25,15 @@
...
-    "@aurodesignsystem/webcorestylesheets": "^4.3.1"
+    "@aurodesignsystem/design-tokens": "^4.1.1",
+    "@aurodesignsystem/webcorestylesheets": "^5.0.2"
   },
   "devDependencies": {
     "@alaskaairux/design-tokens": "^3.14.0",
```
Comment (blackfalcon) [inline package.json]: Remove legacy package reference

File: src/style-toaster.scss [hunk 4]
```diff
@@ -4,27 +4,30 @@
 // ---------------------------------------------------------------------
 
 // Import Auro tokens
-@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
-@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties';
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables';
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SassCustomProperties';
```
Comment (blackfalcon) [inline src/style-toaster.scss]:
```suggestion
```

File: src/style-toaster.scss [hunk 5]
```diff
@@ -4,27 +4,30 @@
...
 
-@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/breakpoints';
-@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/core';
-@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/utilityClasses/displayProperties';
+// Support for fallback values
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables.scss';
```
Comment (blackfalcon) [inline src/style-toaster.scss]:
```suggestion
```

Remove duplicate reference

File: src/style.scss [hunk 6]
```diff
@@ -4,55 +4,58 @@
 // ---------------------------------------------------------------------
 
 // Import Auro tokens
-@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
-@import './../node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties';
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables';
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SassCustomProperties';
```
Comment (blackfalcon) [inline src/style.scss]:
```suggestion
```

File: src/style.scss [hunk 7]
```diff
@@ -4,55 +4,58 @@
...
 
-@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/breakpoints';
-@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/core';
-@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/utilityClasses/displayProperties';
+// Support for fallback values
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables.scss';
```
Comment (blackfalcon) [inline src/style.scss]:
```suggestion
```

**Review by blackfalcon — CHANGES_REQUESTED** [review]
Please update all npm dependencies and update repo to support Node support policy https://auro.alaskaair.com/node-support#gsc.tab=0

File: demo/index.html (hunk 1 above)
Comment (irma-kurnia-alaska) [reply demo/index.html]: ok, sure. thank you.

**Pull request merged**